piff/gp_interp.py

```diff
@@ -141,6 +142,36 @@ def initialize(self, stars, logger=None):
         :param stars:   A list of Star instances to interpolate between
         :param logger:  A logger object for logging debug info. [default: None]
         """
+        if self.rows is None:
+            self.nparams = len(stars[0].fit.params)
+            self.rows = np.arange(0, self.nparams, 1).astype(int)
+        else:
+            self.nparams = len(self.rows)
+
+        if len(self.kernel_template)==1:
+            self.kernels = [self.kernel_template[0] for i in range(self.nparams)]
+        else:
```

Comment:
This is uncovered.  I think it's impossible with only allowing kernel to be a str, given line 95.
Which maybe implies that we don't need it to be a list.

Reply:
If kernel is a string at the beginning, it will be only a list with one kernel inside so it will still be used here